pandas/_libs/src/period_helper.h

```diff
@@ -108,12 +108,9 @@ typedef npy_int64 (*freq_conv_func)(npy_int64, asfreq_info *af_info);
  * new pandas API helper functions here
  */
 
-npy_int64 asfreq(npy_int64 period_ordinal, int freq1, int freq2, char relation);
-
 freq_conv_func get_asfreq_func(int fromFreq, int toFreq);
-void get_asfreq_info(int fromFreq, int toFreq, char relation,
-                     asfreq_info *af_info);
 
 npy_int64 get_daytime_conversion_factor(int from_index, int to_index);
+int max_value(int a, int b);
```

Comment:
really? ok, need to fix this up later. too much duplicated floating around, but I guess when this is fully cythonized can remove this.